torch/_torch_docs.py

```diff
@@ -3427,7 +3427,48 @@ def merge_dicts(*dicts):
 .. math::
     \Gamma(\text{input}_i) = \int_0^\infty t^{(\text{input}_i-1)} e^{-t} dt.
 
-See :func:`torch.lgamma` for a related function.
+See :func:`torch.igammac` and :func:`torch.lgamma` for related functions.
```

Comment:
Docs look great and appear consistent with igamma, but the example needs to be updated to use igammac instead of igamma. Want to show that igammac + igamma = 1 in the example, too?

Reply:
Sure